docs/utilities/data_classes.md

```diff
@@ -1180,7 +1181,25 @@ AWS Secrets Manager rotation uses an AWS Lambda function to update the secret. [
             do_something_with(record.body)
     ```
 
-### VPC Lattice
+### VPC Lattice V2
```

Comment:
Suggestion - keep without the v2 to not break deep linking any customer might have had.

The default will be v2 then, and V1 gets a new anchor